config/formats.js

```diff
@@ -562,6 +562,36 @@ let Formats = [
 		ruleset: ['Pokemon', 'Standard NEXT', 'Team Preview'],
 		banlist: ['Uber'],
 	},
+	{
+		name: '[Gen 7] Shitmons',
+		threads: [
+			`&bullet; <a href="http://spo.ink/shitmons">Shitmons</a>`,
+		],
+
+		gameType: 'doubles',
+		teamLength: {
+			validate: [2, 2],
+			battle: 2,
+		},
+		ruleset: ['Cancel Mod', 'HP Percentage Mod'],
+		banlist: [
+			'Honedge', 'Doublade', 'Aegislash', 'Kitsunoh', 'Shedinja ++ Sturdy', 'Power Construct', 'Schooling', 'Iron Barbs', 'Rough Skin',
+			'Pure Power', 'Huge Power', 'Cursed Body', 'Moody', 'Pressure', 'Wonder Guard', 'Fur Coat', 'Parental Bond', 'Protean', 'Stamina',
+			'Snow Warning', 'Sand Stream', 'Primordial Sea', 'Desolate Land', 'Poison Heal', 'Ice Body', 'Rain Dish', 'Grassy Surge',
+			'Cheek Pouch', 'Aguav Berry', 'Berry', 'Berry Juice', 'Black Sludge', 'Enigma Berry', 'Figy Berry', 'Iapapa Berry', 'Leftovers',
+			'Mago Berry', 'Oran Berry', 'Sitrus Berry', 'Wiki Berry', 'Rocky Helmet', 'Normalium-Z', 'Aerodactylite', 'Aggronite', 'Ampharosite',
+			'Blastoisinite', 'Blazikenite', 'Charizardite X', 'Charizardite Y', 'Diancite', 'Galladite', 'Garchompite', 'Gardevoirite',
+			'Gyaradosite', 'Latiasite', 'Latiosite', 'Lucarionite', 'Metagrossite', 'Salamencite', 'Sceptilite', 'Steelixite', 'Swampertite',
+			'Tyranitarite', 'Venusaurite',
+		],
+		onValidateSet: function (set) {
+			let template = this.getTemplate(set.species);
```

Comment:
I would declare `problems` above this (`let problems =[];`)

Reply:
All the `problems` suggestions breaks convention with every other validated format in the file.

Reply:
Yeah but without `problems` it requires you to revalidate after each fix, so if you have multiple issues it will take a while to fix your team.

Reply:
Mix and Mega is validated in the exact same way, one problem at a time.

Reply:
I know, but i was planning on fixing that another time.